solutions/event-routing-account-settings/catalogValidationValues.json.template

```diff
@@ -0,0 +1,3 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY"
```

Comment:
missed this typo - fixed in https://github.com/terraform-ibm-modules/terraform-ibm-activity-tracker/pull/60